### ISSUE TITLE
Consider exposing System.getCanonical during builds

### ISSUE THREAD

**Issue by guybedford** [systemjs/builder#289]
This would allow a function like:

```javascript
System.getCanonical('http://site.com/some/module.js'); // -> some/module or whatever
```

That would only be available at build time, making it possible to use in plugins for example.

//cc @EisenbergEffect

**Comment by guybedford** [comment]
Note this is now `System.getCanonicalName` so we're consistent with `builder.getCanonicalName` (https://github.com/systemjs/builder/commit/d7bfb5fa6b98017f51d125612696a52463e2cec6).

**Comment by Alxandr** [comment]
Question, why only at build time btw? Or rather, what is "build time" in this case. Will it be available in the browser when my plugin does `translate`? Or only during bundling?

**Comment by guybedford** [comment]
It would only be available during bundling yes.

**Comment by guybedford** [comment]
Released in 0.13.5.
